src/Translatable/Translatable.php

```diff
@@ -32,6 +34,12 @@ public static function bootTranslatable(): void
             /* @var Translatable $model */
             return $model->saveTranslations();
         });
+
+        static::deleted(function (Model $model) {
```

Comment:
I would wrap this call in the if to don't add the listener if there is no need to.

Reply:
I think it's better to leave it as it is, since you can enable or disable cascade deletion after retrieving a model from database. In this case, the already gathered model will not have the listener updated